### ISSUE TITLE
Paper checkbox checked color has an extra parenthesis added to it.

### ISSUE THREAD

**Issue by hankish** [PolymerElements/paper-checkbox#164]
### Description

The `--paper-checkbox-checked-color` property is getting corrupted.

**The markup:**
```
  <paper-checkbox checked="{{item.selected}}" id="checkbox" class$="{{cardClass}}">
  </paper-checkbox>
```

**The css:**
```
  paper-checkbox {
    --paper-checkbox-size: 1.3em;
    --paper-checkbox-checked-color: #f57c00;
  }
```

After selecting the checkbox in the actual page, here is Chrome's inspected view of the CSS on the `paper-checkbox #checkboxContainer #checkbox` div:
```
  .paper-checkbox-1 #checkbox.checked.paper-checkbox {
      background-color: #f57c00);
      border-color: #f57c00);
  }
```

NOTE: The extra parenthesis on each line. That is getting added automatically somehow and it is then getting ignored by Chrome as invalid.

Any ideas?

**Comment by bicknellr** [comment]
Wow, I completely missed this; are you still being affected by it? This looks like it was a problem with Polymer's CSS rewriting for shady DOM, but [it doesn't seem to repro now](http://jsbin.com/nevahus/2/edit?html,output). (Or maybe my example excludes too much?)

**Comment by hankish** [comment]
Yes we're still experiencing the issue, but to be honest, we're stuck on an outdated version of the component so at this point it's probably our fault. :)

You can probably just close this.

Thanks for following up though!

**Comment by bicknellr** [comment]
What version of Polymer are you using? Also, if you have a reduced test case, I'd be up for digging a little further.
